src/tsconfig.json

```diff
@@ -1,7 +1,14 @@
 {
   "compilerOptions": {
-    "target": "ES5",
+    "allowJs": true,
+    "checkJs": true,
+    "lib": ["es2018", "dom"],
+    "jsx": "react",
+    "jsxFactory": "createElement",
     "module": "commonjs",
-    "allowJs": true
-  }
+    "noEmit": true
+  },
+  "include": [
+    "sidebar/util/*.js"
+  ]
 }
```

Comment:
The reference for this config file is here: https://www.typescriptlang.org/docs/handbook/tsconfig-json.html